server/app/index.js

```diff
@@ -19,6 +19,8 @@ app.use(express.json());
 
 app.use(express.static(staticPath));
 
+app.use('/api/beverages', require('../api/beverages'));
```

Comment:
Switch to app.use('/api', require('../api/index.js')).
Then mount beverages in /api/index.js